src/dnsimple/Services/Registrar.cs

```diff
@@ -100,7 +100,7 @@ public RegistrarService(IClient client) : base(client)
         /// <see cref="DomainTransfer"/>
         /// <see>https://developer.dnsimple.com/v2/registrar/#transferDomain</see>
         public SimpleDnsimpleResponse<DomainTransfer> TransferDomain(long accountId,
-            string domainName, DomainTransfer transfer)
+            string domainName, DomainTransferInput transfer)
```

Comment:
Given the change, wouldn't it make more sense to change the argument name to `transferInput` instead of `transfer` ?

Reply:
I think it's reasonable.